examples/hybrid/cli_options.jl

```diff
@@ -89,10 +89,14 @@ function parse_commandline()
         help = "Energy variable name [`rhoe` (default), `rhoe_int` , `rhotheta`]"
         arg_type = String
         default = "rhoe"
-        "--upwinding"
-        help = "Upwinding mode [`none` (default), `first_order` , `third_order`]"
-        arg_type = String
-        default = "none"
+        "--energy_upwinding"
+        help = "Energy upwinding mode [`none` (default), `first_order` , `third_order`, `boris_book`, `zalesak`]"
```

Comment:
I think Tapio mentioned that we don't need BB at all in CA. It was an intermediate operator that was needed for us to understand how to set up these FCT as CC operators and to have a baseline comparison for QA

Reply:
Might as well leave it in if it's just a few extra lines of code? If it's removed from ClimaCore, I'll get rid of it here.